package.json

```diff
@@ -190,6 +190,7 @@
     "mssql-v10": "npm:mssql@^10.0.4",
     "mysql": "^2.18.1",
     "mysql2": "^3.11.3",
+    "nan": "^2.22.0",
```

Comment:
I tried installing event-loop-stats etc on an empty project. It worked with v23-pre.

So I figured out it was caused by an older `nan` dependency in our lock file.
A sub dependency of e.g. rdfkafka.

https://github.com/nodejs/nan/blob/main/CHANGELOG.md